src/main/java/seedu/address/logic/commands/DeleteStaffCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_PROJECT_DONT_EXIST;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_STAFF_NAME;
+
+import java.util.List;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.project.ProjectName;
+import seedu.address.model.staff.Staff;
+import seedu.address.model.staff.StaffName;
+import seedu.address.model.staff.UniqueStaffList;
+
+/**
+ * Delete a staff from a project.
+ */
+public class DeleteStaffCommand extends Command {
+    public static final String COMMAND_WORD = "delstaff";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the project identified by the index number used in the displayed project list.\n"
+            + "Parameters: "
+            + PREFIX_PROJECT_NAME + "PROJECT NAME "
+            + PREFIX_STAFF_NAME + "STAFF NAME\n"
+            + "Example: " + COMMAND_WORD + " pn/CS2103 sn/Shawn Law";
+
+    public static final String MESSAGE_DELETE_STAFF_SUCCESS = "Deleted Staff: %1$s";
+
+    private String projectName;
+
+    private String staffName;
+
+    /**
+     * Creates a DeleteStaffCommand.
+     * @param staffName the staff name to check for
+     * @param projectName the project name to check for
+     */
+    public DeleteStaffCommand(StaffName staffName, ProjectName projectName) {
+        this.staffName = staffName.staffName;
+        this.projectName = projectName.fullName;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Project> projectList = model.getFilteredProjectList();
+        int len = projectList.size();
+        Project project = null;
+        for (int x = 0; x < len; x++) {
+            String nameToCheck = projectList.get(x).getProjectName().fullName;
```

Comment:
Instead of calling fullName attribute, can call toString() instead to maintain abstraction?

Reply:
Noted and changed